sdk/ai/azopenai/models.go

```diff
@@ -1105,6 +1122,9 @@ type ChatCompletionsOptions struct {
 	// A collection of textual sequences that will end completions generation.
 	Stop []string
 
+	// Options for streaming response. Only set this when you set stream: true.
```

Comment:
StreamOptions should be moved to GetChatCompletionsStreamOptions - we provide two separate functions where one of them enables streaming and the other doesn't and this option doesn't apply to the non-streaming mode.

Underneath it does use these structures, but we don't expose that to the user.